libqaul/src/lib.rs

```diff
@@ -2,19 +2,19 @@
 
 mod auth;
 mod crypto;
-mod users;
+pub mod users;
```

Comment:
Actually I think maybe we keep the module private, then use fascading to expose only what people need (i.e. `User` and `UserData`. Considering there'll be more stuff in `users`, this feels more elegant and we can have a simple `api` prelude or something)

Reply:
Definitely. I'll expose only those structs here.